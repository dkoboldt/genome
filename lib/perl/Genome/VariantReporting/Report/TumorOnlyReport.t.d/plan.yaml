reports:
    "tumor-only":
        interpreters:
            position: {}
            vep: {}
            rsid: {}
            caf: {}
            nhlbi: {}
            "1kg": {}
            vaf:
                sample_names:
                    - sample_name
