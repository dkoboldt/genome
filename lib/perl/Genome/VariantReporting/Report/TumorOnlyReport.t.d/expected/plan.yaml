---
experts: {}
needs_translation: 1
reports:
  tumor-only:
    filters: {}
    interpreters:
      1kg: {}
      caf: {}
      nhlbi: {}
      position: {}
      rsid: {}
      vaf:
        sample_names:
          - sample_name
      vep: {}
